packages/app/src/server/routes/api/cortex/index.ts

```diff
@@ -31,6 +32,34 @@ export const cortexProxy = httpProxy.createProxyServer({ ignorePath: true });
  */
 cortexProxy.on("proxyReq", onProxyReq);
 
+cortexProxy.on("proxyRes", (proxyRes, req, res) => {
+  const isHTMLResponse = proxyRes.headers["content-type"]?.includes("text/html")
+  let data: any = undefined;
+  const body: Array<any> = [];
+  res.writeHead(proxyRes.statusCode!, proxyRes.headers);
```

Comment:
I am not 100% if just taking the headers from the response might have any unexpected side-effects. It worked fine during testing in this case, but if anyone knows more, please let me know!